firefox-ios/Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -232,8 +232,7 @@ class SearchViewController: SiteTableViewController,
 
     /// Whether to show non-sponsored suggestions from Firefox Suggest.
     private var shouldShowNonSponsoredSuggestions: Bool {
-        viewModel.isPrivate ?
-        model.shouldShowPrivateModeFirefoxSuggestions :
+        !viewModel.isPrivate &&
         model.shouldShowFirefoxSuggestions
```

Comment:
```suggestion
        return !viewModel.isPrivate && model.shouldShowFirefoxSuggestions
```

we should be using returns (we agreed on that as a team, because it's easier for quick scanning legibility). Also, theres' no reason why this && needs to be on a separate line.